test/parallel/test-zlib-from-concatenated-gzip.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Test unzipping a gzip file that contains multiple concatenated "members"
+
+require('../common');
+const assert = require('assert');
+const zlib = require('zlib');
+
+const data = Buffer.concat([
+  zlib.gzipSync('abc'),
+  zlib.gzipSync('def')
+]);
+
+assert.equal(zlib.gunzipSync(data).toString(), 'abcdef');
+
+zlib.gunzip(data, (_, result) => {
```

Comment:
do we need to wrap this kind of callback in `common.mustCall` to make sure the assertions are really executed?